build.go

```diff
@@ -136,7 +272,8 @@ Environment Variables:
 	}
 	BUILDDATE := time.Now().UTC().Format(time.RFC3339)
 
-	GOVERSION := cmd("go", "version")
+	GOVERSION := runtime.Version() //cmd("go", "version")
```

Comment:
man, I should look at these better. This one was mine, too. Just forgot to delete it. Sorry.